src/docs/development/packages-and-plugins/developing-packages.md

```diff
@@ -268,10 +273,10 @@ you can specify the iOS language using `-i` and the
 Android language using `-a`. For example:
 
 ```terminal
-$ flutter create --template=plugin -i objc hello
+$ flutter create --template=plugin --platforms=android,ios -i objc hello
 ```
 ```terminal
-$ flutter create --template=plugin -a java hello
+$ flutter create --template=plugin --platforms=android,ios -a java hello
```

Comment:
So, ALL of these examples use Android and iOS. Could we mix it up a bit?

Reply:
By reading the paragraph above, I thought these examples are to teach users to choose programming languages for android and iOS. So in that case, we should only use Android and iOS here.
However, we can add some examples to the above paragraph that explains the `--platforms` flag. Those examples can have different platforms just to explain the flag. WDYT?

Reply:
Oops. Yeah, that's what I get for just looking at the trees, not the forest.